### DISCUSSION TITLE
Active state not working (for me) in tailwind JIT

### DISCUSSION THREAD

**Discussion by Miguelklappes** [tailwindlabs/tailwindcss#3812] [Help]
Hello people,

Basically the title. From what I've seen in Adam's video, all variants are enabled out of the box, so I'm not sure why it's not taking the styles, can somebody here give me a hint?

I'm using Vue, Vite and Tailwind JIT (along with the typical things like PostCSS, autoprefixer, etc)

**Comment by adamwathan** [comment]
Hey! Can you provide some code that reproduces the issue? Should "just work" so it's hard to know what to suggest without seeing the code 🤔

**Reply by Miguelklappes** [reply]
Yes, I apologize

Although I’m not sure what code to provide; the template, it’s just a simple drop-down menu with these classes:

`<li class="menu-item has-submenu flex flex-row gap-2 items-center relative pb-4 active:text-brand-500“ aria-has-popup=”true“ aria-expanded=”false“>`

[This is the tailwind.config.js and postcss.config.js](https://github.com/Miguelklappes/test)
